database/src/lib.rs

```diff
@@ -315,11 +315,9 @@ pub trait ScyllaStorageManager {
 
     async fn prepare_query(
         scylla_db_session: &std::sync::Arc<scylla::Session>,
-        query_text: &str,
+        mut query: scylla::statement::query::Query,
```

Comment:
Could you share your reasoning on why did you make it mutable?

Reply:
In this method we are establishing consistency and for that we need the query to be mutable.